lib/features/email/domain/usecases/calendar_event_maybe_interactor.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:core/presentation/state/failure.dart';
+import 'package:core/presentation/state/success.dart';
+import 'package:dartz/dartz.dart';
+import 'package:jmap_dart_client/jmap/account_id.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:tmail_ui_user/features/email/domain/repository/calendar_event_repository.dart';
+import 'package:tmail_ui_user/features/email/domain/state/calendar_event_maybe_state.dart';
+
+class MaybeCalendarEventInteractor {
```

Comment:
The file name should be consistent with the class name

Reply:
Done